### ISSUE TITLE
Mapping words to images

### ISSUE THREAD

**Issue by Ram-N** [Ram-N/av-reader#1]
The process of taking 2-3 words and find the right image file is non-trivial.

Step 1: Take the trigram. For each of the 3 words, get the set of image names. (Choose the one with the maximum frequency)
Step 2: See if any of the image rows have "attributes" that match the words.
Ex. "blue hat" might match "blue.jpg", "hat.jpg". But if there is a hat_blue.jpg, that is the best.

3: For each image file, need to create a set of "Attributes" (These could be a list)

Basically, the word_image_map needs to get a lot more sophisticated.
